plugin/evm/vm_test.go

```diff
@@ -364,53 +365,55 @@ func testBuildEthTxBlock(t *testing.T, scheme string) {
 		configJSON:  getConfig(scheme, `"pruning-enabled":true`),
 	})
 
+	vm := tvm.vm
```

Comment:
I tried to align this test as closely with the coreth version of possible, hence the increased diff. It is not exactly possible, as there are a number of vm utilities in coreth that have not been synced to subnet-evm.